pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -334,16 +334,23 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 	// Create a component handler to manage the rendered component.
 	handler := utils.NewComponentHandler(log, r.client, r.scheme, policyRecommendation)
 
+	// Determine the namespaces to which we must bind the cluster role.
+	bindNamespaces, err := helper.TenantNamespaces(r.client)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
```

Comment:
I think we will need a clause like this:

```go
if tenant.MultiTenant() {
  bindNamespaces = append(bindNamespaces, render.PolicyRecommendationNamespace)
}
```

To support impersonation.

Reply:
This isn't quite what I meant - here's an example for the manager controller:

https://github.com/tigera/operator/blob/713ed247a1d1f0e921970d03d620a0bb6f700f65/pkg/controller/manager/manager_controller.go#L622-L632

Reply:
That said, you have handled this differently here: https://github.com/tigera/operator/pull/3052/files#diff-28bf4bd22da8886729331acba1bb2f7c4e423502ac4aed05ce35ac05e1e8e257R240-R281

So I think there's no need to do _both_ of these - you can leave this how it was before if you are already granting those permissions via that special binding.

```go
		bindNamespaces, err := helper.TenantNamespaces(r.client)
		if err != nil {
			return reconcile.Result{}, err
		}
```